### PR TITLE
ftr: Add left/right arrow buttons on desktop view only

### LINKED ISSUE #17: 💡 [REQUEST] - Add left/right arrow buttons on desktop view only
### Start Date

_No response_

### Implementation PR

_No response_

### Reference Issues

_No response_

### Summary

Need to implement a new feature to improve the navigation experience on desktop devices. The goal is to add left and right arrow buttons for navigating through stories in desktop view. These buttons will allow users to move to the previous or next story using visible, clickable controls.

Details:

Arrow Buttons: Left and right arrow buttons should be visible only on desktop view. On mobile and tablet devices, the buttons should remain hidden.
Hiding/Disabling Taps: Currently, the user can navigate between stories by tapping on the left or right side of the screen. With the introduction of arrow buttons, the left/right taps for navigation should be disabled or hidden when the app is viewed on a desktop device. This ensures navigation is fully controlled by the newly added arrow buttons.

Acceptance Criteria:

Arrow buttons should only be visible on desktop view.
Left/right taps for story navigation should be disabled or hidden on desktop.
Mobile and tablet devices should continue using tap-based navigation without any changes.
This feature aims to provide a more user-friendly navigation experience for desktop users.

### Basic Example

_No response_

### Drawbacks

_No response_

### Unresolved questions

_No response_

### PR DESCRIPTION
**Pull request by meetbhalodi11**
Changes:
added left right navigation buttons for desktop view
added media query for showing those buttons on desktop view only (min screen size: 1024px)

## Checklist before requesting a review
- [o] I have performed a self-review of my code.
- [o] Followed the repository's [Contributing Guidelines](https://github.com/Gauravdarkslayer/ngx-stories/blob/main/CONTRIBUTING.md).
- [o] I ran the app and tested it locally to verify that it works as expected.
- [o] I have starred the repository.

Closes #17

### CODE REVIEW

**Comment by Gauravdarkslayer** [comment]
Hey @meetbhalodi11 ,
Thanks for raising this valuable PR, and the implementation looks good as well.
One thing i would like to suggest from design perspective is currently the next & previous buttons are not visible properly, so can you update the icons similar to this https://dwello.in/webstory/12-prime-residential-projects-in-mumbai ?

**Comment by meetbhalodi11** [comment]
Looks great. We can do that

File: projects/ngx-stories/src/lib/ngx-stories.component.html [hunk 1]
```diff
@@ -39,5 +39,14 @@ <h5 [style.visibility]="isHolding ? 'hidden' : 'visible'">
   <button class="prev" (click)="navigateStory('previous')">Previous</button>
   }
   <button class="next" (click)="navigateStory('next')">Next</button>
+
+
+  <!-- Next button -->
+  <div class="desktop-navigation next-btn" role="button" tabindex="0" (click)="navigateStory('next')"></div>
```
Comment (Gauravdarkslayer) [inline projects/ngx-stories/src/lib/ngx-stories.component.html:45]: Any particular reason that we are not using `<button>` tag here ?
Just want to know.

**Review by Gauravdarkslayer — COMMENTED** [review]
Comment for button added

File: projects/ngx-stories/src/lib/ngx-stories.component.html (hunk 1 above)
Comment (meetbhalodi11) [reply projects/ngx-stories/src/lib/ngx-stories.component.html:45]: My bad. We can have button there

**Review by Gauravdarkslayer — APPROVED** [review]

**Comment by Gauravdarkslayer** [comment]
Thanks for doing the change 🎉  @meetbhalodi11

**Pull request merged**